lib/logger.ts

```diff
@@ -54,6 +54,8 @@ export class Logger {
   static set(config: Config): void {
     if (config.troubleshoot) {
       Logger.logLevel = LogLevel.DEBUG;
+    } else if (config.logLevel) {
+      Logger.logLevel = (<any>LogLevel)[config.logLevel];
```

Comment:
I guess you figured out but it is used to convert string to enum, found on [SO](http://stackoverflow.com/questions/17380845/how-to-convert-string-to-enum-in-typescript). 

Would you prefer a comment to make it easier to read or just remove it for now?